applovin_max/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## Versions
 
-## x.x.x
+## 4.2.2
 * Update `MaxAd` to include `adFormat`, `networkPlacement`, and `latencyMills`.
 * Depends on Android SDK v13.1.0 and iOS SDK v13.1.0.
```

Comment:
Historically - did we match version bump with that of the native SDK version bump? Since minor version bump, should it be minor version bump for plugin as well?

Reply:
Also, added some new APIs so that should be minor version bump at the very least.

Reply:
Bump to 4.3.0.  Will submit another PR.